README.md

```diff
@@ -284,6 +284,44 @@ src\Support\Traits\LastModifiedBy.php
 ```
 
 
+# ActivityLoggable Trait
+
+The `ActivityLoggable` trait provides functionality for logging activities and communication logs related to a model.
+ 
+Methods
+-------
+
+*   **`getActivityLogEmails(): array`**: Get the emails associated with activity logs.
+*   **`activities(): Collection`**: Get the collection of activities associated with the model.
+*   **`getModelChanges(?array $modelChangesJson = null): string`**: Get the model changes as a formatted string.
+*   **`getModelChangesJson(bool $allowCustomAttribute = false): array`**: Get the model changes as an array of JSON. If `$allowCustomAttribute` is date return of `getModelChanges` the activity will only track the of mention in `getModelChanges` 
+*   **`prepareModelChange($attribute, $from, $to): array`**: Prepare the model change data.
```

Comment:
I have no idea what the three parameters here are and what I am meant to pass to this. When do you use this? Before `createActivityLog` ?

Reply:
Thank you, it was redundant, I just deleted it